helm/prometheus-rules/templates/alerting-rules/prometheus.rules.yml

```diff
@@ -10,6 +10,25 @@ spec:
   groups:
   - name: prometheus
     rules:
+    - alert: PrometheusAvailabilityRatio
```

Comment:
I guess this is a temporary alert until we have an slo alert right? Can you add an ops-recipe link even if it's empty?

Reply:
Well, I put the `prometheus-resource-limit-reached` opsrecipe because it explains quite well how to understand and fix all the reasons why prometheus would die.
That's better than an empty opsrecipe I guess?

Reply:
Also, regarding SLOs:
I could have found a way to have an "SLO burn rate" alert. But I don't like them for a few reasons:
- they are quite generic and point to a generic opsrecipe
- the promql query is harder to understand

As I already had a nice query for this ratio in the availability dashboard, I thought I could as well use it.

Proper solution would be to use SLOTH, but it's not ready yet.

Reply:
I agree with all your points, because even if we have an slo alert, each should have their different opsrecipe. I was thinking the expression behind your alert would be a good first step for our SLO